spec/spec_helper.rb

```diff
@@ -3,15 +3,10 @@
 $LOAD_PATH.unshift File.expand_path("../lib", __FILE__)
 
 require 'bundler'
-Bundler.setup(:default, :test)
+Bundler.require(:test)
 
-
-require "effin_utf8"
 require "bunny"
-require "rabbitmq/http/client"
-
 
-require "amq/protocol/version"
 puts "Using Ruby #{RUBY_VERSION}, amq-protocol #{AMQ::Protocol::VERSION}"
```

Comment:
Maybe it's more relevant to show SHA of the `amq-protocol` master branch here instead of version.

Reply:
What is there is no "SHA" because we don't run with a local clone?

Reply:
We can do that in `.travis.yml`.